### PR TITLE
javascript3-week2/NicoQ

### PR DESCRIPTION
**Pull request by lnknfan90**
Added the HW, thanks for your review  :)

### CODE REVIEW

File: 3_javascript/javascript3/week2/CurrencyCalculator/script.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    () => (toCurrency = toCurrencySelect.value)
+  );
+
+  const apiKey = "1bd1c44ffdf9e866b3f41d32";
+  let allData = "";
+  let fromCurrency = "EUR";
+  let toCurrency = "DKK";
+  let amount = 0;
+
+  const currencies = [
```
Comment (balance3840) [inline 3_javascript/javascript3/week2/CurrencyCalculator/script.js:31]: You already have a list of all the currencies in the response of the API, so you could build this currency array from the API there. Visit this link for reference [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)

File: 3_javascript/javascript3/week2/CurrencyCalculator/script.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+function domListener() {
+  document.addEventListener("DOMContentLoaded", domLoader);
+}
+
+function domLoader() {
+  const inputAmount = document.getElementById("amount");
+  const clickConvert = document.getElementById("convert");
+
+  clickConvert.addEventListener("click", getData);
+  inputAmount.addEventListener("input", () => (amount = inputAmount.value));
```
Comment (balance3840) [inline 3_javascript/javascript3/week2/CurrencyCalculator/script.js:11]: Nice use of the callback functions here.

File: 3_javascript/javascript3/week2/CurrencyCalculator/script.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+          resolve(data);
+        })
+        .then(() => conversionResult());
+    });
+  }
+
+  function conversionResult() {
+    if (isNaN(amount)) {
+      const printResult = document.getElementById("conversionResult");
+      printResult.innerText = `Please enter the amount of money \n(use a period  "." as the decimal point)`;
```
Comment (balance3840) [inline 3_javascript/javascript3/week2/CurrencyCalculator/script.js:230]: Nice idea to give feedback to the user when something goes wrong.

File: 3_javascript/javascript3/week2/CurrencyCalculator/script.js (hunk 1 above)
Comment (lnknfan90) [reply 3_javascript/javascript3/week2/CurrencyCalculator/script.js]: Thanks for the suggestion!  I made the change taking the currencies from the API

**Pull request closed without merging**